static-content-hosting/src/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link href="<static-website-url>style.css" rel="stylesheet" type="text/css" data-preload="true"/>
```

Comment:
I assume you've tested, but wanted to make sure that `/` is always part of the `$STATIC_WEBSITE_URL`

Reply:
all tested and working!